docs/source/cuda/bindings.rst

```diff
@@ -4,28 +4,40 @@ CUDA Bindings
 Numba supports two bindings to the CUDA Driver APIs: its own internal bindings
 based on ctypes, and the official `NVIDIA CUDA Python bindings
 <https://nvidia.github.io/cuda-python/>`_. Functionality is equivalent between
-the two bindings, with two exceptions:
-
-* the NVIDIA bindings presently do not support Per-Thread Default Streams
-  (PTDS), and an exception will be raised on import if PTDS is enabled along
-  with the NVIDIA bindings.
-* The profiling APIs are not available with the NVIDIA bindings.
+the two bindings.
```

Comment:
Side note... should this section on bindings refer to the cuda overview doc on how to install/obtain the NVIDIA bindings? As part of review I looked at the rendered RTD docs and thought perhaps this section ought to link through to how to get them.

Reply:
There's a link in the first sentence to the docs for CUDA Python, which begins with how to install them: https://nvidia.github.io/cuda-python/

Reply:
Yes :) I found that after writing the comment! I'm not sure it's evident that the installation instructions are at the linked location but am happy to leave for now.